deploy/example_catalog/cr-app-cassandra-3.11.json

```diff
@@ -67,9 +66,8 @@
                 }
             }
         ],
-        "setup_package": {
-            "config_api_version": 7,
-            "import_url": "https://s3.amazonaws.com/bluek8s/kubedirector/cassandra311/appconfig-3.1.tgz"
+        "config_package_url": {
```

Comment:
should be config_package

Reply:
I am changing all the other apps now and testing ... looks like I hit a roadblock. Gimme some time to get back to you.